synthtool/gcp/templates/ruby_library/AUTHENTICATION.md

```diff
@@ -0,0 +1,179 @@
+# Authentication
+
+In general, the {{ metadata["package_name"] }} library uses [Service
+Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+credentials to connect to Google Cloud services. When running on Compute Engine
+the credentials will be discovered automatically. When running on other
+environments, the Service Account credentials can be specified by providing the
+path to the [JSON
+keyfile](https://cloud.google.com/iam/docs/managing-service-account-keys) for
+the account (or the JSON itself) in environment variables. Additionally, Cloud
```

Comment:
could "environment variables" link to the section below

Reply:
Done.